README.md

```diff
@@ -31,51 +31,89 @@ The docker-compose recipe is set up so that you can use the containers for devel
 
+The MySQL image available on DockerHub uses MySQL's recommended configs for production which causes mysqld to eat up ~500MB when the container is started. Make sure that the system running the container has more than 1GB of RAM available.
+``
+Building the image
+```
+docker build -t cami/store:1.0 -f docker\cami-store\Dockerfile .
+```
 Run the store container:
 ```
-docker run -d --name cami-store -P cami/store:1.0
+docker run -d --hostname cami-store --name cami-store -P cami/store:1.0
+```
+If you're not running the database in a VM, then you need to obtain the local port that redirects to the docker container's `3306` (default mysql port). To obtain it run the command:
 ```
+$ docker ps -l
+```
+And find the corresponding local port redirecting to 3306 from the output (e.g. 0.0.0.0:`32774`->3306) `[1]`
 
 ### cami-rabbitmq
-First build the image(wihth credentials builtin).
+This instance hosts a Rabbit MQ server that will be used as a message broker by all **cami** components. The provisioning script installs the RabbitMQ instance, creates a `cami` vhost and a `cami` user
+with full permissions on that vhost.
+
+Default ports for Rabbit MQ server are `15672` (for accessing the web console) amd `5672` for the amqp protocol used to send messages.
+
+First build the image (the credentials are builtin).
 ```
 docker build -t cami/rabbitmq:1.0 -f docker/cami-rabbitmq/Dockerfile .
 ```
-
-Run a container. We need to specify the hostname since it si usde by rabbitmq
-nodes to identify themselves.
+Run a container. We need to specify the hostname since it is used by rabbitmq nodes to identify themselves.
 ```
 docker run -d --hostname cami-rabbitmq --name cami-rabbitmq -P cami/rabbitmq:1.0
 ```
 
-Get the port on which we can acces the rabbitmq management interface. In this
- case the default port was is
-```
-vagrant@docker:~/cami-project$ docker ps -l
-CONTAINER ID        IMAGE               COMMAND                  CREATED             STATUS              PORTS                                                                                                                                                     NAMES
-47101a9ced98        cami/rabbitmq:1.0   "docker-entrypoint.sh"   4 seconds ago       Up 2 seconds        0.0.0.0:32775->4369/tcp, 0.0.0.0:32774->5671/tcp, 0.0.0.0:32773->5672/tcp, 0.0.0.0:32772->15671/tcp, 0.0.0.0:32771->15672/tcp, 0.0.0.0:32770->25672/tcp   cami-rabbitmq
-```
-
-Access the management interface.
+Get the corresponding local port on which we can acces the rabbitmq management interface and the amqp protocol. 
 ```
-http://127.0.0.1:32771
+$ docker ps -l
 ```
+In this case we search for the entries `0.0.0.0:32778->5672/tcp` and `0.0.0.0:32776->15672/tcp`, meaning that:
+- to access the rabbitmq web admin console we use `http://{IP}:32776`
```

Comment:
@paraschivionut for simplicity purposes, can we investigate if it's possible to have a fixed IP? Or a DNS setting? This may be a bit cumbersome. I would put it as an issue in the backlog